### ISSUE TITLE
Highlight other changesets in the area when viewing/editing a changeset

### ISSUE THREAD

**Issue by stevage** [mapillary/mapillary_issues#830]
![screenshot 2015-03-19 10 07 10](https://cloud.githubusercontent.com/assets/678344/6721277/cbdbca96-ce1f-11e4-83df-3e2ba27837d4.png)

This kind of email says very little, and if you have requested more than one change, it's essentially meaningless.

Suggestions: 
- include the date of relevant photos, and the date (in human "yesterday evening" terms) the request was made
- use reverse geocoded locations
- use a thumbnail of an image
- use an image of the relevant part of the map
- mention what changes were requested

The current email is basically just useless spam.

**Comment by polarbearing** [comment]
Not only the approval mail is insufficient, the whole process of changeset creation is very unsystematic for the user. 

I had to correct angles for the 10 recent sequences. I was able (at least I hope so) to combine 2 sequences in one changeset because they were the only tracks on this road. Once there are more tracks in parallel, it is impossible to find the one to correct for further editing in the same changeset. 

I have no indication for which of the 10 sequences I now have created the changeset, if they all have the correct angle values, or if one of them is missing or requested twice. 

From the ca 8 changesets, only one is shown (so far) in activities and confirmed by mail.
